examples/rmrk/resource/wasm/tests/resources.rs

```diff
@@ -373,4 +464,55 @@ impl<'a> Fixture<'a> {
         .concat();
         program.send_bytes(actor_id, encoded_request)
     }
+
+    async fn add_resource_using_client(
+        &'a self,
+        actor_id: ActorId,
+        resource_id: u8,
+        resource: rmrk_resource_client::Resource,
+    ) -> Result<(u8, rmrk_resource_client::Resource), TestError> {
+        let mut resource_client = self.resource_client();
+        let call = resource_client
+            .add_resource_entry(resource_id, resource)
+            .with_args(GTestArgs::new(self.admin_id.into()))
+            .publish(actor_id)
```

Comment:
Let's rename the `actor_id` param everywhere as it confuses. I guess this what tripped you up in the first place